qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,261 @@
+
+
+class QVExperiment(BaseExperiment):
+    """RB Experiment class"""
+
+    # Analysis class for experiment
+    __analysis_class__ = QVAnalysis
+
+    # ExperimentData class for the simulations
+    __simulation_data__ = ExperimentData
+
+    _trials = 0
+
+    def __init__(
+        self,
+        qubits: Union[int, Iterable[int]],
+        trials: Optional[int] = 1,
+        seed: Optional[Union[int, Generator]] = None,
+        simulation_backend: Optional[BaseBackend] = None,
+    ):
+        """Standard randomized benchmarking experiment
+        Args:
+            qubits: the number of qubits or list of
+                    physical qubits for the experiment.
+            trials: number of trials to run the quantum volume circuit.
+            seed: Seed or generator object for random number
+                  generation. If None default_rng will be used.
+            simulation_backend: the simulator backend to use to generate
+                the expected results. the simulator must have a 'save_probabilities' method.
+                if None Aer simulator will be used (in case Aer is not installed - qiskit-info
+                will be used).
+        """
+        if not isinstance(seed, Generator):
+            self._rng = default_rng(seed=seed)
+        else:
+            self._rng = seed
+        self._trials = trials
+        self._previous_trials = 0
+        if not simulation_backend and HAS_SIMULATION_BACKEND:
+            self._simulation_backend = Aer.get_backend("aer_simulator")
+        else:
+            self._simulation_backend = simulation_backend
+        super().__init__(qubits)
+
+    def run(
+        self,
+        backend: "Backend",
+        analysis: bool = True,
+        experiment_data: Optional[ExperimentData] = None,
+        **kwargs,
+    ):
+        """Run an experiment and perform analysis.
+        Args:
+            backend (Backend): The backend to run the experiment on.
+            analysis: If True run analysis on experiment data.
+            experiment_data (ExperimentData): Optional, add results to existing
+                experiment data. If None a new ExperimentData object will be
+                returned.
+            kwargs: keyword arguments for self.circuit,
+                    qiskit.transpile, and backend.run.
+        Returns:
+            ExperimentData: the experiment data object.
+            tuple: If ``return_figures=True`` the output is a pair
+                   ``(ExperimentData, figures)`` where ``figures`` is a list of figures.
+        """
+        # Create new experiment data
+        if experiment_data is None:
+            experiment_data = self.__experiment_data__(self)
+        else:
+            # count the number of previous trails.
+            # assuming that all the data in experiment data is QV data.
+            # divide by 2 (because for each trial there is also simulation data)
+            self._previous_trials = int(len(experiment_data.data()) / 2)
```

Comment:
I changed the experiment to have 2 experiment data lists.
the experiment run now returns only the real backend data, and the simulation data is stored at the experiment object in case the user will want to add additional trials